### PR TITLE
A post should have only one markdown H1

### PR DESCRIPTION
**Pull request by mishig25**
A post should have only one markdown H1, which is its title. All other sections should start with H2 or H3 etc.

### CODE REVIEW

**Review by pacman100 — APPROVED** [review]
Thank you @mishig25 for all the work in fixing the blog posts! 🤗

File: deploy-tfserving-kubernetes.md [hunk 1]
```diff
@@ -11,8 +11,6 @@ authors:
 # Deploying 🤗 ViT on Kubernetes with TF Serving
 
 
-# Introduction
-
```
Comment (pcuenca) [inline deploy-tfserving-kubernetes.md:15]: +1

**Review by julien-c — APPROVED** [review]
lgtm, but out of curiosity, what issue was this creating?

**Comment by mishig25** [comment]
there was no technical issue. But "look-wise", it felt "wrong" to see huge h1 titles everywhere in the middle of an article like 


image

**Comment by julien-c** [comment]
makes sense, thanks!

**Pull request merged**